ocaml/testsuite/tests/typing-modes/modes.ml

```diff
@@ -0,0 +1,59 @@
+(* TEST
+   * expect *)
```

Comment:
I think there should be a test that shows what happens if you specify the local mode in both syntaxes at once. My expectation is that the type-checker will complain that locality was specified twice.

Reply:
Unfortunately, `let x @ local local = ...` is parsed as mode coercion `let x = [%jane.modes local local] ...` which lacks duplication check. I just made some changes so it's detected.